content/pt/docs/4.directory-structure/12.store.md

```diff
@@ -173,22 +188,22 @@ export default {
 }
 ```
 
-E as ações (actions) correspondentes podem estar no ficheiro `store/actions.js`:
+E as ações correspondentes pode estar dentro do ficheiro `store/actions.js`:
 
 ```js{}[store/actions.js]
 export default {
   async fetchCounter({ state }) {
-    // fazer requisição
-    const response = { data: 10 };
-    state.counter = response.data;
-    return response.data;
+    // faça um pedido
+    const res = { data: 10 };
+    state.counter = res.data;
+    return res.data;
   }
 }
 ```
 
 ### Exemplo de estrutura de pasta
 
-Uma configuração complexa da estrutura de ficheiro/pasta memória pode parecer com isto:
+Uma configuração complexa de estrutura de ficheiro ou pasta da emória pode se parecer com isto:
```

Comment:
`memória`

Reply:
@ChristopheCVB thanks for noting that, I have the habit write quickly,
and some words go out with some characters missing.